### DISCUSSION TITLE
Javadoc / documentation suggestion as the single sentence

### DISCUSSION THREAD

**Discussion by novoj** [community/community#59090] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'm using the IntelliJ Idea Copilot integration and when I write a JavaDoc comment or Markdown documentation Copilot always suggests the whole line, which often consists of ending the current sentence and starting a new one. I often find myself in a situation where I'd accept the end of the current sentence, but reject the start of the next one. So I have to tab to accept the entire proposal and then manually delete the start of the next sentence, which is usually invalid. I think it would be helpful if Copilot only suggested the end of the current sentence, which ends with a period. If it's accepted, it can suggest the beginning of the next sentence, but as a follow-up suggestion, so that the user is not forced to delete it manually, but can easily decide not to take advantage of it and start typing a different beginning.

**Comment by jasonbronson** [comment]
Yeah happens to me as well. I think it's broken
